splunk-otel-android/src/test/java/com/splunk/rum/SessionIdChangeTracerTest.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Splunk Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.splunk.rum;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import java.util.List;
+
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.trace.Tracer;
+import io.opentelemetry.sdk.testing.junit4.OpenTelemetryRule;
+import io.opentelemetry.sdk.trace.data.SpanData;
+
+public class SessionIdChangeTracerTest {
+    @Rule
+    public OpenTelemetryRule otelTesting = OpenTelemetryRule.create();
+
+    private SessionIdChangeListener underTest;
+
+    @Before
+    public void setup() {
+        Tracer tracer = otelTesting.getOpenTelemetry().getTracer("testTracer");
+        underTest = new SessionIdChangeTracer(tracer);
+    }
+
+    @Test
+    public void shouldEmitSessionIdChangeSpan() {
```

Comment:
can you add a test to verify that nothing bad happens if there is no listener hooked up?

Reply:
whoops. this comment should be on the SessionIdTest.

Reply:
never mind. I don't think there is anything new needed.